src/routes/competitions/smt-2026/+page.svelte

```diff
@@ -314,8 +314,7 @@
       <p>
         Students are highly encouraged to participate in SMT 2026 as part of
         their <b>school team</b>. Students whose school is sending a team to SMT
-        must attend with their school. Homeschooled students are considered to
-        be a part of their local public school.
+        must attend with their school. 
```

Comment:
Are homeschooled students now allowed to go anywhere? This should prolly be clarified somewhere if its being removed